.goreleaser.yml

```
@@ -1,33 +1,48 @@
 # You can find the GoReleaser documentation at http://goreleaser.com
 project_name: exercism
 
+env:
+  - CGO_ENABLED=0
 builds:
-- env:
-    - CGO_ENABLED=0
-  main: ./exercism/main.go
-  goos:
-    - darwin
-    - linux
-    - windows
-    - freebsd
-    - openbsd
-  goarch:
-    - amd64
-    - 386
-    - arm
-    - arm64
-    - ppc64
-  goarm:
-    - 5
-    - 6
-  ignore:
-    - goos: openbsd
-      goarch: arm
-    - goos: freebsd
-      goarch: arm
-
-checksum:
-  name_template: '{{ .ProjectName }}_checksums.txt'
+  - id: release-build
+    main: ./exercism/main.go
+    mod_timestamp: '{{ .CommitTimestamp  }}'
+    flags:
+      - -trimpath # removes file system paths from compiled executable
+    ldflags:
+      - '-s -w' # strip debug symbols and DWARF debugging info
+    goos:
+      - darwin
+      - linux
+      - windows
+      - freebsd
+      - openbsd
+    goarch:
+      - amd64
+      - 386
+      - arm
+      - arm64
+      - ppc64
+    goarm:
+      - 5
+      - 6
+    ignore:
+      - goos: openbsd
+        goarch: arm
+      - goos: freebsd
+        goarch: arm
+  - id: installer-build
+    main: ./exercism/main.go
+    mod_timestamp: '{{ .CommitTimestamp  }}'
+    flags:
+      - -trimpath # removes file system paths from compiled executable
+    ldflags:
+      - '-s -w -X {{ .ModulePath }}/cmd.Version={{.Version}}' # sets binary version to match tag without 'v' prefix
```

Oops missed this one. 

```suggestion
      - '-s -w' # strip debug symbols and DWARF debugging info
```